src/Utils/normalization.jl

```diff
@@ -130,21 +111,27 @@ for CAC_S in (:CACCS, :CACDS)
     end
 end
 
+@inline isidentity(B::IdentityMultiple) = B.M.λ == oneunit(B.M.λ)
```

Comment:
Why not [`one`](https://github.com/JuliaLang/julia/blob/8c0b550b61bcb196e1c8a9b03b38f0e6eb8a9dd7/base/number.jl#L297-L298), which is more efficient than [`oneunit`](https://github.com/JuliaLang/julia/blob/8c0b550b61bcb196e1c8a9b03b38f0e6eb8a9dd7/base/number.jl#L320-L321) and equivalent for scalars?

Reply:
Thanks for the links to the implementation of these two methods. Ok, let's use `one`. If we eventually use dimensions (such as ODE models with units with `Unitful`), we may switch back to `oneunit`.

> more efficient

both calls super fast (fractions of ns).

Reply:
I was just wondering why you changed it. I thought there was some deeper reason behind. We can keep it if you want.

Reply:
The difference is that `oneunit` refers to the identity for a given type, e.g.

```julia
julia> m = I(2.0u"m/s", 2)
IdentityMultiple{Quantity{Float64,𝐋*𝐓^-1,Unitful.FreeUnits{(m, s^-1),𝐋*𝐓^-1,nothing}}} of value 2.0 m s^-1 and order 2

julia> oneunit(m.M.λ)  # the type doesn't change
1.0 m s^-1

julia> one(m.M.λ)   # the type is now Number
1.0
```